packages/application-shell/src/components/navbar/navbar.spec.js

```diff
@@ -634,6 +638,50 @@ describe('rendering', () => {
               hideDashboard: false,
             },
             permissions: [permissions.ViewProducts],
+            keyOfMenuItem: 'products',
+          };
+          wrapper = shallow(
+            <RestrictedMenuItem {...props}>
+              <ItemChild />
+            </RestrictedMenuItem>
+          );
+        });
+        it('should match snapshot', () => {
+          expect(wrapper).toMatchSnapshot();
+        });
+        it('should render <RestrictedByPermissions>', () => {
+          expect(wrapper).toRender(RestrictedByPermissions);
+        });
+      });
+    });
+    describe('disabled menu items', () => {
+      describe('when passed menu item should be disabled', () => {
+        beforeEach(() => {
+          props = {
+            menuVisibilities: {},
+            keyOfMenuItem: 'orders',
+            disabledMenuTiems: ['orders'],
+          };
+          wrapper = shallow(
+            <RestrictedMenuItem {...props}>
+              <ItemChild />
+            </RestrictedMenuItem>
+          );
+        });
+        it('should not render <ToggleFeature>', () => {
+          expect(wrapper).not.toRender(ToggleFeature);
+        });
+        it('should not render <RestrictedByPermissions>', () => {
+          expect(wrapper).not.toRender(RestrictedByPermissions);
+        });
+      });
+      describe('when passed but menu item should not be disabled', () => {
```

Comment:
One bored day I will rewrite this suite to RTL.

Reply:
Yeah, actually we have a "low priority" task to improve and rewrite the navbar component...I was hoping then to rewrite the tests when we do that 😇

Reply:
It would be kind of dope to rewrite the tests before to have a hardened suite and then just rewrite the NavBar against the existing test suite.

Reply:
Right, even better then!